src/tagged-type.ts

```diff
@@ -14,7 +14,26 @@ import {
   EmbeddedSet,
   StreamToken,
 } from "./values";
-import { QueryValueObject, QueryValue } from "./wire-protocol";
+import {
+  QueryValue,
+  QueryInterpolation,
+  ObjectFragment,
+  ArrayFragment,
+  FQLFragment,
+  ValueFragment,
+  TaggedType,
+  TaggedLong,
+  TaggedInt,
+  TaggedDouble,
+  TaggedObject,
+  EncodedObject,
+  TaggedTime,
+  TaggedDate,
+  TaggedMod,
+  TaggedRef,
+  TaggedBytes,
+} from "./wire-protocol";
+import { Query, QueryArgument, QueryArgumentObject } from "./query-builder";
```

Comment:
I had some reorganization of types staged but I put everything back. I was worried about 
1. breaking import paths, and
2. complicating the diff for this PR.

On further consideration of no. 1, that doesn't actually matter because we bundle the driver and only expose the exports from src/index and that's it.

So I guess my point is: agreed! We should come back to this.